packages/cicero-core/src/metadata.js

```diff
@@ -316,6 +344,42 @@ class Metadata {
         return this.packageJson.name + '@' + this.packageJson.version;
     }
 
+    /**
+     * Returns the dimension of the image
+     * @param {Buffer} buffer buffer of the image
+     * @param {String} mimeType the mime-type of image
+     * @returns {Object} the dimension of the image
+     */
+    static getImageDimensions(buffer, mimeType) {
+        if(mimeType === 'image/png') {
```

Comment:
I would wrap a `try ... catch` around this code. We could be in a situation where: the binary file looks like it has a png header, but it has nothing else so those reads could throw an error. (e.g., for a corrupted png).